### DISCUSSION TITLE
Open Source or Not？

### DISCUSSION THREAD

**Discussion by orbitgw** [nigels-com/glew#346] [Q&A] [answered]
I just read the license.
It's said that"**The automatic code generation scripts are released under the GNU GPL.**".
Does it means I should open the source of my project?

**Comment by nigels-com** [comment, marked as answer]
I can tell you my understanding of the licensing, but also link to the original wording
which has long been associated with the project.

There are two aspects to the license.

The GLEW library the provides the runtime API as BSD or MIT style (not GPL) license.
Therefore including the GLEW header(s) or linking the GLEW library would ordinarily be BSD or MIT licensed.
This would be the majority of GLEW uses, I expect.

The scripts used to produce the GLEW library source code are GPL licensed.
Forking the code generation scripts would be GPL licensed.

See also:
https://github.com/nigels-com/glew#copyright-and-licensing
